adapters/adagio/adagiotest/supplemental/ext-bid-adagio.json

```diff
@@ -0,0 +1,165 @@
+{
+  "mockBidRequest": {
+    "test": 1,
+    "id": "some-request-id",
+    "device": {
+      "ua": "test-user-agent",
+      "ip": "123.123.123.123",
+      "language": "en",
+      "dnt": 0
+    },
+    "tmax": 1000,
+    "user": {
+      "buyeruid": "awesome-user"
+    },
+    "site": {
+      "page": "test.com",
+      "publisher": {
+        "id": "123456789"
+      }
+    },
+    "imp": [
+      {
+        "id": "some-impression-id",
+        "banner": {
+          "w":320,
+          "h":50
+        },
+        "ext": {
+          "bidder": {
+            "organizationId": "1000",
+            "site": "site-name",
+            "placement": "ban_atf"
```

Comment:
How is this test case conceptually different from `banner-web.json`?

Reply:
This is basically the same, I created this test because the `adagio` bid extension is not always filled. But you right, I removed this test and moved the extension inside the video-web.json test which is the real case: when we return a video imp, we fill the extension.